### ISSUE TITLE
The package chokes with an older avy version

### ISSUE THREAD

**Issue by dotemacs** [mrkkrp/zzz-to-char#1]
tl;dr when using this package with an older version of `avy` the package does not work

See how I stumbled upon this issue:

<kbd>M-x</kbd> package-install <kbd>RET</kbd> zzz-to-char <kbd>RET</kbd>
Then in a buffer with some text:
```
foo█ x x x x bar
```
<kbd>M-x</kbd> zzz-up-to-char <kbd>RET</kbd>

In minibuffer:
```
Zzz up to char:
```
<kbd>b</kbd>
I get back (in minibuffer):
> Symbol’s function definition is void: avy-jump

Mind you I had `avy` at version `20180514.1100`.

Installed the newest version from MELPA.
And voila, it worked.

At first I thought:

> Maybe you'd like to specify the minimum `avy` version to something higher that `0.3.0` in order to prevent this?

But then looking at `avy` at version `20180514.1100`, even that package is at version `0.4.0`. So it won't be a fix.

Figured you might wanna know about this...

Thanks for the package :)

**Comment by mrkkrp** [comment]
Yes, this is how it works in Emacs/MELPA world. Glad you got it working, I think we can close the issue because it doesn’t have any meaningful resolution.
